loci.ttl

```diff
@@ -198,50 +198,3 @@ This ontology constrains both normative instruction on what constitutes LocI obj
   skos:definition "Something (likely a prov:Entity) had a method (prov:Plan) that was used to generate it"@en ;
   skos:editorialNote "This property is a property chain axiome equivalent to a chian of three properties in the PROV ontology: prov:wasGeneratedBy, prov:qualifiedAssociation and prof:hadPlan linking a prov:Entity (the thing generated) to a prov:Plan (the thin instruction generation - the method) via a prov:Activity (the generating event) and a prov:Association that the prov:Activity has linking it to the prov:Plan."@en ;
 .
-
-
-<http://catalogue.linked.data.gov.au/org/abs>
```

Comment:
Why also removing orgs?

Reply:
Because this isn't an exhaustive list of organisations. This is a list that should be maintained outside of the ontology? Each time we cross a new organisation, we dont want to update this file.

Reply:
The links (like http://catalogue.linked.data.gov.au/org/csiro) all work. I think the goal of caching them here was to have human-readable labels to hand

Reply:
i agree with @shaneseaton on this. the issue we had here is that there is a listing of loci datasets, but that didn't match with the list being loaded into a cache. that should come from a register of datasets or a specialised list for each cache (there may be multiple).

Using that line of reasoning, that would extend to lists of any individuals like organisation. lists of organisations should be maintained in a different file.

that would allow loci.ttl to be imported everywhere.